services/core/WeatherAgent/config

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is an invalid config file...note the comment on region means that the block will not end.

Also, when testing this with just a zip I was unable to get results published to the bus.

Reply:
Fixed the config file and readme.